nucleus/pom.xml

```diff
@@ -276,6 +276,7 @@
         <jackson.version>2.9.4</jackson.version>
         <!-- A StAX implementation for JSON. -->
 
+        <javaee.api.version>8.0</javaee.api.version>
```

Comment:
Would be more clear to move this up a few lines so its directly underneath the "java ee" comment. (here: https://github.com/mulderbaba/Payara/blob/aad23d0ba3fe09dec8f726c46c446b2d2c323244/nucleus/pom.xml#L221)